region_migrations/run.go

```diff
@@ -0,0 +1,127 @@
+package region_migrations
+
+import (
+	"fmt"
+
+	"github.com/Scalingo/cli/config"
+	scalingo "github.com/Scalingo/go-scalingo"
+	errgo "gopkg.in/errgo.v1"
+)
+
+func Create(app string, destination string, dstAppName string) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get scalingo client")
+	}
+
+	migration, err := c.CreateRegionMigration(app, scalingo.RegionMigrationParams{
+		Destination: destination,
+		DstAppName:  dstAppName,
+	})
+	if err != nil {
+		return errgo.Notef(err, "fail to create migration")
+	}
+
+	err = c.RunRegionMigrationStep(app, migration.ID, scalingo.RegionMigrationStepPreflight)
+	if err != nil {
+		return errgo.Notef(err, "fail to run preflight step")
+	}
+
+	err = WatchMigration(c, app, migration.ID, RefreshOpts{
+		ExpectedStatuses: []scalingo.RegionMigrationStatus{
+			scalingo.RegionMigrationStatusPreflightError,
+			scalingo.RegionMigrationStatusPreflightSuccess,
+		},
+	})
+	if err != nil {
+		return errgo.Notef(err, "fail to watch migration")
+	}
+
+	return nil
+}
+
+func Run(app, migrationId string, step scalingo.RegionMigrationStep) error {
+	c, err := config.ScalingoClient()
+	if err != nil {
+		return errgo.Notef(err, "fail to get scalingo client")
+	}
+
+	migration, err := c.ShowRegionMigration(app, migrationId)
+	if err != nil {
+		return errgo.Notef(err, "fail to show region migration")
+	}
+
+	shouldContinue := ConfirmStep(migration, step)
+	if !shouldContinue {
+		fmt.Println("The operation has been aborted. You can restart it later.")
+		fmt.Println("If you want to abort the migration, run:")
```

Comment:
The UI is:
```
scalingo -a myapp migration-run MIGRATION_ID --prepare

This will do:
 - A
 - B
 - C

? Continue (y/N) N
The operation has been aborted. You can restart it later.
If you want to abort the migration, run:
scalingo --app myapp migration-abort MIGRATION_ID
```

The user did not abort the migration but just the current step. He can restart it later or abort the migration if he want to abort the entire migration.

Maybe the wording is not right. I'm open to suggestion.

Reply:
Well, I think you said it right in the comment. The user aborted the current step so I suggest: "The current step has been cancelled. You can..." 

That way, we don't use the word "abort". And I find "current step" clearer than "operation". :)